### ISSUE TITLE
[1.12.2] [Suggestion] Bee Grub, Queen Bee and Royal Jelly rebalance

### ISSUE THREAD

**Issue by KorDum** [MatrexsVigil/harvestcraft#111]
Thanks for mod!
Mod have some recipes:

> Bee Grub + Royal Jelly = Queen Bee

> Queen Bee = Royal Jelly

but Queen Bee after done work in apiary produce 2+ Bee Grub and 1 Queen Bee. And the Bee Grub as a result are useless, do you think?
Can you rebalance this recipes? Maybe `1 Glass Bottle + 8 Honey = Royal Jelly` and 1 Bee Grub (instead many Bee Grub and 1 Queen Bee) after queen work done?

**Comment by KorDum** [comment]
Or
```
[
    [Honey, Pressed Wax, Honey],
    [Pressed Wax, Glass Bottle, Pressed Wax],
    [Honey, Pressed Wax, Honey]
]
```

**Comment by MatrexsVigil** [comment]
Fixed in 1.12.2h

EDIT:  You can change the queenbeeitem recipe JSON manually in the recipes folder of the JAR file

- Added: Config: queenbeelastresultequalsQueen [Default: True]: Determines if a queen bee's last result will be another queen bee or a random item (grub, waxcomb, or honey comb)
